A.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="A.css">
+</head>
+<body>
+    <div class="header1">
+        <h1>Hello Microverse!</h1>
```

Comment:
Great job @Tekhlay 🤝

- [ ] [Optional] You may kindly use more descriptive names for your files instead of `A.html` you may use `index.html`.

Reply:
Ok brother, only I made them to follow the instruction of the project? Thanks for your suggestion!